src/test/ui/lint/outer-forbid.stderr

```diff
@@ -28,26 +28,14 @@ LL | #[allow(unused)]
    = note: for more information, see issue #81670 <https://github.com/rust-lang/rust/issues/81670>
 
 error[E0453]: allow(nonstandard_style) incompatible with previous forbid
-  --> $DIR/outer-forbid.rs:30:9
+  --> $DIR/outer-forbid.rs:29:9
    |
 LL | #![forbid(unused, non_snake_case)]
    |                   -------------- `forbid` level set here
 ...
 LL | #[allow(nonstandard_style)]
    |         ^^^^^^^^^^^^^^^^^ overruled by previous forbid
 
-error: allow(unused_variables) incompatible with previous forbid
-  --> $DIR/outer-forbid.rs:20:9
-   |
-LL | #![forbid(unused, non_snake_case)]
-   |           ------ `forbid` level set here
-...
-LL | #[allow(unused_variables)]
-   |         ^^^^^^^^^^^^^^^^ overruled by previous forbid
-   |
-   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
-   = note: for more information, see issue #81670 <https://github.com/rust-lang/rust/issues/81670>
```

Comment:
This seems like a regression to me, the primary span is different for this one, but matches the one in line 26.

Reply:
The diagnostic's primary spans are same, but only with different notes.

Reply:
Ah! You're right. I didn't expand the whole diff to see that there was a prior lint at line 20, confusing it with the one on line 30.